### PR TITLE
docs: support image directory

### PR DESCRIPTION
**Pull request by mcutalo88**
### Description
While developing a recent blog #924, I wanted to add several images to the blog post. In this PR we simply create a new `images` directory that will be parse by the site generator and moved to the destination directory so these assets my be used by anything on the site.

### Testing Performed

local.

### CODE REVIEW

**Comment by danielhochman** [comment]
I'm not quite following here, were images from the static/ directory not usable in blog posts prior to this change?

Also images should always be in the static/ directory so they are properly handled by the build tools.

Last but not least, I think we should probably start using GitHub's hosting by default for blog post images anyways, there is little benefit to keeping them (or any images for that matter) in the repo itself.

File: docs/_website/generator/generate.go [hunk 1]
```diff
@@ -148,6 +148,14 @@ func main() {
 		log.Fatal(err)
 	}
 
+	// Get all images files to be moved.
+	images, err := getFiles(docsRoot, ".png")
```
Comment (danielhochman) [inline docs/_website/generator/generate.go:152]: this results in a fatal error when the png is fed to generate then ParseFiles

Comment (mcutalo88) [reply docs/_website/generator/generate.go:152]: Idk its been working on my feature branch with my blog post.

**Comment by mcutalo88** [comment]
I actually did not notice thats where all the images were I was only focused on the generated output. Will close this pr.

**Pull request closed without merging**